core/src/main/java/org/apache/struts2/components/Set.java

```diff
@@ -83,6 +83,8 @@
 public class Set extends ContextBean {
     protected String scope;
     protected String value;
+    protected boolean doNotTrimBody;
+    protected boolean stripBodyLineBreaks;
```

Comment:
Shouldn't this be set to `true` by default to persist current behaviour?

Reply:
Isn't this `stripBodyLineBreaks` too much specific to make additional attribute in `set` tag?

Reply:
The `doNotTrimBody `was replaced with `trimBody `(set to true by default to mimic current behaviour) in the most recent commit.

The `stripBodyLineBreaks` , in my mind, had more immediate applicability to the `s:set` tag than most others.  I figured it could be made available to the `set` tag initially (a small and relatively safe change), then possibly considered for a more general future refactoring.

Reply:
@JCgH4164838Gh792C124B5 Can you show some use case for removing line breaks?
There is a good reason to introduce not trimming because there is no way to get real value in set tag. But `stripBodyLineBreaks` looks way too specific. It is like replacing `a` letters with `b`. Should we create an attribute for that too? :)
Besides this really should be done in the controller not in the view. And even if you want to do this in the view then you can call `replaceAll` directly in the jsp.

Reply:
Hello @aleksandr-m .
The `stripBodyLineBreaks` was more of a convenience enhancement idea than a necessity.  The use cases for it are not very strong, as you pointed out above.  Given the concerns raised for the `stripBodyLineBreaks` attribute it has been removed in the latest commit - which also simplifies the `getBody()` logic and test changes too.  :)